formats/hocon/build.gradle

```diff
@@ -12,17 +12,9 @@ compileKotlin {
     }
 }
 
-configurations {
-    apiElements {
-        attributes {
-            attribute(TargetJvmVersion.TARGET_JVM_VERSION_ATTRIBUTE, 8)
-        }
-    }
-    runtimeElements {
-        attributes {
-            attribute(TargetJvmVersion.TARGET_JVM_VERSION_ATTRIBUTE, 8)
-        }
-    }
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
```

Comment:
With these options `TARGET_JVM_VERSION_ATTRIBUTE ` is assigned to value `8` in module.json and `configurations` block is not needed.

Also without this, I was unable to update `apiDump` because of errors like this:
```
Symbol is declared in unnamed module which is not read by current module
```
also, there was the warning:
```
'compileJava' task (current target is 11) and 'compileKotlin' task (current target is 1.8) jvm target compatibility should be set to the same Java version.
```